src/codeu/chat/server/Server.java

```diff
@@ -53,10 +54,9 @@
   private final Timeline timeline = new Timeline();
 
   private final Map<Integer, Command> commands = new HashMap<>();
-
   private final Uuid id;
   private final Secret secret;
-
+  private static final ServerInfo info = new ServerInfo();
```

Comment:
This is how the `ServerInfo` is created **on the server side.** You probably want to pass in a parameter here?

Basically, you're going to want to update the server version variable whenever you make a change to the server code. Where you do that is up to you: either by passing a parameter in, or by changing the variable inside the `ServerInfo` class. But you want to do that in a way that's easy to test, too.

So right now you either want to add a parameter here, or you want to change your test class so it doesn't break when people change the variable.

Does that make more sense?